dist/common/limits.d/scylla.conf

```diff
@@ -1,5 +0,0 @@
-scylla  -  core     unlimited
```

Comment:
I don't see an equivalent to this setting.

Reply:
@avikivity Added LimitCORE=infinity to scylla-server.service.